pkg/kind/kind.go

```diff
@@ -303,11 +310,11 @@ containerdConfigPatches:
     endpoint = ["http://%s:5000"]
 nodes:
 - role: control-plane
-  image: %s
+  image: %s %s
```

Comment:
You're wanting to do something like [this](https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts), right? If so, I think you'll need to update the config file to add the extra mounts to the node as described in that link...

Reply:
Thanks for reviewing so quickly!

Yup! That is exactly what I was wanting to do with `extraMounts`

Hmm it worked when I tested it. Do you encounter an error?

This new `%s` I added is passed the `extraMount` string which is:

```\nextraMounts:
  - hostPath: %s
    containerPath: %s
```

So I think the final bit resolves to:

```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
name: %s
containerdConfigPatches:
- |-
  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:%s"]
    endpoint = ["http://%s:5000"]
nodes:
- role: control-plane
  image: %s
  extraMounts:
  - hostPath: %s
    containerPath: %s
  extraPortMappings:
  - containerPort: 31080
    listenAddress: 0.0.0.0
    hostPort: 80`
```

with all the `%s` replaced with the proper variables. 

Is there some other config I should be updating?

Reply:
No, that's my bad ... I exec'ed into the wrong container :facepalm: